.github/workflows/release.yaml

```diff
@@ -66,12 +81,19 @@ jobs:
           docker push ${{ secrets.ACR_LOGIN_SERVER }}/scalardb-server:${{ steps.version.outputs.version }}
           docker push ${{ secrets.ACR_LOGIN_SERVER }}/scalardb-server:${{ steps.version.outputs.minor_version }}
 
+      # Push the latest tag image since it is required from the perspective of the specification of Azure Marketplace.
       - name: Push latest tag to Azure Container Registry
         if: ${{ steps.version.outputs.version == steps.version.outputs.latest_version }}
         run: |
           docker tag ghcr.io/scalar-labs/scalardb-server:${{ steps.version.outputs.version }} ${{ secrets.ACR_LOGIN_SERVER }}/scalardb-server:latest
           docker push ${{ secrets.ACR_LOGIN_SERVER }}/scalardb-server:latest
 
+      # Push the patch version image only since AWS Marketplace restricts using the latest tag.
+      - name: Push containers to Amazon Elastic Container Registry
```

Comment:
We push the patch version image only since AWS Marketplace restricts using the latest tag.